roles/stunserver/meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
+  author: your name
+  description: your role description
+  company: your company (optional)
```

Comment:
Do we need to fill these fields? Otherwise I would drop them.

Reply:
The meta information is only relevant for ansible galaxy in case we push it there.